src/views/Dashboard/Lab/NavigationLayout.vue

```diff
@@ -160,7 +160,7 @@ export default {
 
     lastEventData(val) {
       if (val !== null) {
-        if (val.method === "LabRegistered") return
+        if (val.method === "LabStakeSuccessful") return
```

Comment:
customer should get notification not based on `LabStakeSuccessful` event, but after trigger send email verification

Reply:
notification already appears when sending email, you can see from the origin branch in this PR

https://github.com/azmimuhammad/debio-frontend/blob/8ac5dc1215a7ab80592c5f1e92fc2c876a9862a3/src/views/Dashboard/Lab/Registration/Services/index.vue#L465